ecommerce/pols/templates/edx/checkout/receipt.html

```diff
@@ -22,6 +22,7 @@
 {% endblock navbar %}
 
 {% block content %}
+  {{ custom_settings.GOOGLE_TAG_MANAGER_SNIPPET|safe }}
```

Comment:
Safe check for the attribute.
```suggestion
  {{ getattr(custom_settings, 'GOOGLE_TAG_MANAGER_SNIPPET', '') | safe }}
```